src/SerilogTracing/Enrichers/ElapsedMilliseconds.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Serilog.Core;
+using Serilog.Events;
+using SerilogTracing.Interop;
+using Constants = SerilogTracing.Core.Constants;
+
+namespace SerilogTracing.Enrichers;
+
+/// <summary>
+/// An enricher that adds the span duration as milliseconds.
+/// </summary>
+public class ElapsedMilliseconds: ILogEventEnricher
```

Comment:
These should probably be internal, we've ended up doing that across the board in Serilog projects, unless a type is really useful as a building block - makes it easier to refactor and improve the implementations down the track.